bgpq4_printer.c

```diff
@@ -772,6 +772,8 @@ bgpq4_print_aspath(FILE* f, struct bgpq_expander* b)
 		return bgpq4_print_nokia_md_aspath(f, b);
 	case V_HUAWEI:
 		return bgpq4_print_huawei_aspath(f, b);
+	case V_ARISTA:
+		return bgpq4_print_cisco_aspath(f, b);
```

Comment:
I'm happy to create a distinct `bgpq4_print_arista_aspath` and `bgpq4_print_arista_oaspath `, but it seemed like it might be overkill for the purposes of this pull request.